js_modules/dagit/packages/core/src/ticks/SensorDryRunDialog.tsx

```diff
@@ -368,11 +374,21 @@ export const EVALUATE_SENSOR_MUTATION = gql`
           error {
             ...PythonErrorFragment
           }
+          dynamicPartitionsRequests {
+            ...DynamicPartitionRequestFragment
+          }
         }
       }
       ...PythonErrorFragment
     }
   }
+
+  fragment DynamicPartitionRequestFragment on DynamicPartitionRequest {
+    partitionKeys
+    partitionsDefName
+    type
+  }
```

Comment:
Not sure why this is redefined here - I think this should be a reference to `${DYNAMIC_PARTITION_REQUESTS_FRAGMENT}`?